lib/pages/new_group/new_group.dart

```diff
@@ -259,10 +261,13 @@ class NewGroupController extends State<NewGroup>
     numberSelectedImagesNotifier.addListener(() {
       if (numberSelectedImagesNotifier.value == 1) {
         Navigator.pop(context);
-        uploadAvatarNewGroupChat(
-          matrixClient: Matrix.of(context).client,
-          entity: imagePickerController.selectedAssets.first.asset
-        );
+        avatarNotifier.value = imagePickerController.selectedAssets.first.asset;
+        removeAllImageSelected();
+        // FIXME: Consider to remove it
+        // uploadAvatarNewGroupChat(
+        //   matrixClient: Matrix.of(context).client,
+        //   entity: imagePickerController.selectedAssets.first.asset
+        // );
```

Comment:
please delete if it is not necessary

Reply:
@hoangdat  every code comments were fixed in next commit